src/components/Guest.js

```diff
@@ -70,18 +70,59 @@ export default function Guest({ personName, guestTitle, guestSocials, children,
 
 export function slugifyGuestName(string) {
   // Approach borrowed from here: https://mhagemann.medium.com/the-ultimate-way-to-slugify-a-url-string-in-javascript-b8e4a0d849e1
-  
-  const a = 'àáâäæãåāăąçćčđďèéêëēėęěğǵḧîïíīįìıİłḿñńǹňôöòóœøōõőṕŕřßśšşșťțûüùúūǘůűųẃẍÿýžźż·/_,:;'
-  const b = 'aaaaaaaaaacccddeeeeeeeegghiiiiiiiilmnnnnoooooooooprrsssssttuuuuuuuuuwxyyzzz------'
-  const p = new RegExp(a.split('').join('|'), 'g')
 
-  return string.toString().toLowerCase()
-    .replace(/\s+/g, '') // Replace spaces with nothing
+  const a =
+    "àáâäæãåāăąçćčđďèéêëēėęěğǵḧîïíīįìıİłḿñńǹňôöòóœøōõőṕŕřßśšşșťțûüùúūǘůűųẃẍÿýžźż·/_,:;"
+  const b =
+    "aaaaaaaaaacccddeeeeeeeegghiiiiiiiilmnnnnoooooooooprrsssssttuuuuuuuuuwxyyzzz-"
+  const p = new RegExp(a.split("").join("|"), "g")
+
+  return string
+    .toString()
+    .toLowerCase()
+    .replace(/\s+/g, "-") // Replace spaces with dash
+    .replace(p, c => b.charAt(a.indexOf(c))) // Replace special characters
+    .replace(/&/g, "-and-") // Replace & with 'and'
+    .replace(/[^\w\-]+/g, "") // Remove all non-word characters
+    .replace(/\-\-+/g, "-") // Replace multiple - with single -
+    .replace(/^-+/, "") // Trim - from start of text
+    .replace(/-+$/, "") // Trim - from end of text
+}
+
+export function slugifyGuestNameForAvatars(string) {
+  // Approach borrowed from here: https://mhagemann.medium.com/the-ultimate-way-to-slugify-a-url-string-in-javascript-b8e4a0d849e1
+  const a =
+    "àáâäæãåāăąçćčđďèéêëēėęěğǵḧîïíīįìıİłḿñńǹňôöòóœøōõőṕŕřßśšşșťțûüùúūǘůűųẃẍÿýžźż·/_,:;"
+  const b =
+    "aaaaaaaaaacccddeeeeeeeegghiiiiiiiilmnnnnoooooooooprrsssssttuuuuuuuuuwxyyzzz-"
+  const p = new RegExp(a.split("").join("|"), "g")
+  if (string === "Amina du Jean") {
+    return "amina-du-jean"
+  }
+  if (string === "Megan Shipman") {
+    return "megan-shipman"
+  }
+  if (string === "Eli Ebberts") {
+    return "eli-ebberts"
+  }
+  if (string === "Kaho Shibuya") {
+    return "kaho-shibuya"
+  }
+  if (string === "Non Sweet") {
+    return "non-sweet"
+  }
```

Comment:
This was basically a last resort thing. I normally wouldn't do this but I didn't want to spend more time fixing it if I made it work somehow.

Reply:
I can understand that - however I think we really want to avoid having hacky code like this in the codebase. We don't have any plans to replace the site right now, and we do have new features lined up -- so this would live for a while. Things like this tend to come back and surprise you a year or two down the road, when you start relying on the function for something new